migrations/20181102102836_add_sliderId_column_to_slider_table.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function (knex, Promise) {
+    return knex.schema
+        .table('slider', function (table) {
+            table.integer('slider_id');
```

Comment:
why do we need this?
the table already has ID column as primary key, lets use that instead